torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -20,14 +19,7 @@
 aten = torch.ops.aten
 
 
-if torch.version.hip:
-
-    def _scaled_dot_product_attention(*args, **kwargs):
-        with sdpa_kernel(backends=[SDPBackend.MATH, SDPBackend.FLASH_ATTENTION]):
-            return aten.scaled_dot_product_attention(*args, **kwargs)
-
-else:
-    _scaled_dot_product_attention = aten.scaled_dot_product_attention
+_scaled_dot_product_attention = aten.scaled_dot_product_attention
```

Comment:
efficient/cudnn attn was disabled previously for the same reason?

Reply:
I believe we were hitting this issue as well as others when ROCm efficient attention support was added, so we opted out of efficient attention in fx_passes to avoid regressions (relevant PR https://github.com/pytorch/pytorch/pull/124885)